src/main/java/chess/controller/ChessGameController.java

```diff
@@ -0,0 +1,28 @@
+package chess.controller;
+
+import chess.controller.command.Command;
+import chess.domain.ChessGame;
+import chess.view.InputView;
+import chess.view.OutputView;
+
+public class ChessGameController {
+
+    private final InputView inputView;
+    private final OutputView outputView;
+
+    public ChessGameController(InputView inputView, OutputView outputView) {
+        this.inputView = inputView;
+        this.outputView = outputView;
+    }
+
+    public void run() {
+        outputView.printStartMessage();
+        ChessGame chessGame = new ChessGame();
+
+        while (!chessGame.isEnd()) {
+            final String rawInputCommand = inputView.inputCommand();
```

Comment:
```suggestion
            final String command = inputView.inputCommand();
```
변수명을 디테일하게 작성할 필요는 없습니다.
클라이언트 입장에서 raw input 이라는 의미를 알아야할 필요가 있을까요?

Reply:
`메서드내 파라미터`에서 뿐만 아니라 `controller 등의 클라이언트 스코프?`내에서도 구체적으로&지엽적으로 변수명을 잡으면 안되겠네요!

- 혹시 메인로직이 돌아가는 내부에서는 원하는 변수명을 지엽적으로 잡아도 될지도 궁금합니다! 제 생각은 yes.. 캡슐화되어서 클라이언트가 모르니까?? 팀원들도 알아먹을 수 있을 정도는 괜찮다?입니다.